### PR TITLE
Update getting started

### PR DESCRIPTION
**Pull request by oponder**
Things have changed a bit since we wrote the getting started part of the README.

This updates that.

It also removes passage and mailcatcher from the dockerfile, I will move it to the dockerfile in api instead, so we don't have to think about starting another docker-compose before testing anything related to invitations. 

FYI @ograu. I gave you some instructions on how to start `passage`, but now I am removing that from `happa`. So what I said here will no longer work after this PR is merged. Instead I will add it to the docker-compose in `api` so that it is always running, and invitations are always working.

https://github.com/giantswarm/happa/pull/712#issuecomment-521104734

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,16 +45,6 @@ make up
 As part of `make up`, `./fixtures.sh` will run and create the initial user and
 organization you can log in with.
```
Comment (marians) [inline README.md]: I am still missing two pieces of info:

1. The docker-compose stuff happens in the `testing` sub folder of giantswarm/api
2. You have to run `make aws|azure|kvm` to create a docke-compose.yaml file.

**Comment by marians** [comment]
Very good to clean this up before the next people are hit! ❤️

**Comment by oponder** [comment]
> Very good to clean this up before the next people are hit! ❤️

Yeah I'm kinda in a "lets make dev smoother" mode.. which conflicts a bit with other prios, but I think it might be one the best places to put effort, since more people are coming on board.

File: README.md (hunk 1 above)
Comment (oponder) [reply README.md]: Ok! adding that.

We do have a `cd` command as part of the example. but i'll add the `make aws` part there.

**Pull request merged**